### PR TITLE
WIP: More Features for Feature 

### PR DESCRIPTION
**Pull request by jnunemaker**
Adding more convenience methods for Feature to make it easier to get what is enabled and what the state of things is.

I'm going to add some convenience methods for enable_actor, enable_group, etc. and their disable counterparts as well. I'll probably also add those to the DSL. I'm also going to move most of the typecasting into the types or GateValue. Going to see how it feels.

fyi @rsanheim @aroben so we don't double work. Hoping to finish this up tomorrow. Feel free to review anything here, though it is not quite final.

### CODE REVIEW

**Comment by jnunemaker** [comment]
Ok, this is good to go I think.

File: README.md [hunk 1]
```diff
@@ -121,6 +129,12 @@ flipper[:stats].enabled? user # false
 # you can enable anything, does not need to be user or person
 flipper[:search].enable group
 flipper[:search].enabled? group
+
+# you can also use shortcut methods
+flipper.enable_actor :search, user
```
Comment (dewski) [inline README.md]: I like these convenience methods :sparkles:

Comment (jnunemaker) [inline README.md]: :cool: :cool: Thought you might.

**Comment by rsanheim** [comment]
:sparkling_heart:

**Pull request merged**